### PR TITLE
Add support for the auto-suggest commands (from Redis Search)

### PR DESCRIPTION
**Pull request by cescoffier**

### CODE REVIEW

**Comment by cescoffier** [comment]
This one was a small group, with only 4 commands. 
The only remaining group from Redis labs is... time series. It contains a lot more commands, so will take more time.

File: extensions/redis-client/runtime/src/main/java/io/quarkus/redis/datasource/search/CreateArgs.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package io.quarkus.redis.datasource.search;
 
-import static io.quarkus.redis.runtime.datasource.Validation.*;
```
Comment (machi1990) [inline extensions/redis-client/runtime/src/main/java/io/quarkus/redis/datasource/search/CreateArgs.java:3]: nice catch.

**Comment by machi1990** [comment]
I started the review on this and then got distracted, I am finishing off the review today. Sorry for the delay @cescoffier

**Review by machi1990 — APPROVED** [review]

**Comment by machi1990** [comment]
lgtm, thanks @cescoffier

**Pull request merged**